tt_metal/hw/firmware/src/brisc.cc

```diff
@@ -406,7 +406,8 @@ int main() {
                     NOC_UNICAST_WRITE_VC,
                     1,
                     31 /*wrap*/,
-                    false /*linked*/);
+                    false /*linked*/,
+                    true /*posted*/);
```

Comment:
Unrelated to your change but a potential bug with the non-posted atomic incs previously: Looking at the additional args we're now setting.. I realize that we never correctly configured the return addr of the non-posted atomic inc.

The pre-incremented value would have been written over the NOC at worker address 0. Seems like we were lucky that this address wasn't being actively used?

This is not an issue with your change, since response traffic for the atomic inc won't be generated anymore.